packages/vuetify/src/components/VTextarea/__tests__/VTextarea.spec.cy.tsx

```diff
@@ -9,24 +9,24 @@ describe('VTextarea', () => {
     const model = ref('Lorem ipsum dolor sit amet consectetur adipisicing elit. ')
     cy.mount(() => <VTextarea auto-grow rows="1" v-model={ model.value } />)
       .get('textarea').first()
-      .should(el => expect(el.outerHeight()).to.equal(56))
+      .should(el => expect(el.outerHeight()).to.equal(58))
```

Comment:
height is increased because input min-height accommodates chip's margin top/bottom